.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"
+    allow:
+      - dependency-name: "moose"
+    schedule:
+      interval: "weekly"
+      time: "01:45"
```

Comment:
So it defaults to Monday if the day is not specified?

Reply:
Yes, the default is monday